src/helpers/storageHelper.js

```diff
@@ -10,6 +10,7 @@ export const storageHelper = {
   globalSettings: () => readFromStore('globalSettings'),
   setGlobalSettings: (globalSettings) => addToStore('globalSettings', globalSettings),
   locationSettings: () => readFromStore('locationSettings'),
+  onboardingSettings: () => readFromStore('onboardingSettings'),
```

Comment:
can you move this one line down please to keep `locationSettings` and `setLocationSettings` next to each other like als the "getter and setter"

Reply:
no i can't. because i removed it instead (was unused)